deobfuscator.py

```diff
@@ -153,15 +149,15 @@ def main(args):
                     # verboseprint(encrypted_strings)
 
                     # Get identifier
-                    match = re.search(r"invoke-static\s+{([vp][0-9]+),", line)
+                    match = re.search(r"invoke-virtual\s+{([vp][0-9]+),", line)
```

Comment:
I don't find any `invoke-virtual` in my code. https://github.com/giacomoferretti/paranoid-deobfuscator/issues/5#issuecomment-1387028926